examples/dump_media/README.md

```diff
@@ -0,0 +1,21 @@
+# Download all media from an account
+
+This script downloads all media in a file / directory hierarchy:
+
+- every user has a separated folder
+- every collection is downloaded into a separate folder
+- media in "My library" goes directly to the user's folder
+- captions (if exists) are next to the media
```

Comment:
`captions (if they exist) are next to the media.`
Somehow we should mention that the caption file name includes the language code.

Reply:
mentioned